wurst/objects/units/Animals/JungleBearDefinition.wurst

```diff
@@ -88,7 +88,7 @@ class AdultJungleBearDefinition extends AdolescentBearDefinition
         this.setScalingValue(1.35)
         this.setSpeedBase(320)
         this.setHitPointsMaximumBase(300)
-        this.setPointValue(600)
+        this.setPointValue(450)
```

Comment:
I legit forgot what was the use of this field, it doesn't seem like it's used at all in Island troll tribes, you mentioned `On top of this the Hp of the adult bear pet has been dropped from 600 -> 450.` did you make a mistake? That's not the HP field, by the way bear pet has been removed in 3.9a if I remember correctly due to a balance issue in case you didn't know.

Reply:
ohh i was not aware of that at all. And yeah I did think that was the unit hp.